### PR TITLE
WT-4889 Dumping additional transaction fields

### PR DESCRIPTION
**Pull request by kommiharibabu**
__wt_verbose_dump_txn_one function is enhanced to dump additional
transaction structure fields for more verbose infomration of the
transaction.

### CODE REVIEW

File: src/txn/txn.c [hunk 1]
```diff
@@ -1627,22 +1627,34 @@ __wt_verbose_dump_txn_one(WT_SESSION_IMPL *session, WT_TXN *txn)
...
+	    "transaction id: %" PRIu64
+	    ", mod count: %u"
 	    ", snap min: %" PRIu64
 	    ", snap max: %" PRIu64
+	    ", no of named snapshots: %u"
 	    ", commit_timestamp: %s"
 	    ", durable_timestamp: %s"
 	    ", first_commit_timestamp: %s"
 	    ", read_timestamp: %s"
+	    ", prepare_timestamp: %s"
```
Comment (sueloverso) [inline src/txn/txn.c]: Alphabetize the timestamps listed, so move this above `read_timestamp`.

File: src/txn/txn.c [hunk 2]
```diff
@@ -1627,22 +1627,34 @@ __wt_verbose_dump_txn_one(WT_SESSION_IMPL *session, WT_TXN *txn)
...
+	    txn->id,
 	    txn->mod_count,
 	    txn->snap_min,
 	    txn->snap_max,
+	    txn->snapshot_count,
 	    __wt_timestamp_to_string(txn->commit_timestamp, ts_string[0]),
 	    __wt_timestamp_to_string(txn->durable_timestamp, ts_string[1]),
 	    __wt_timestamp_to_string(txn->first_commit_timestamp, ts_string[2]),
 	    __wt_timestamp_to_string(txn->read_timestamp, ts_string[3]),
+	    __wt_timestamp_to_string(txn->prepare_timestamp, ts_string[4]),
```
Comment (sueloverso) [inline src/txn/txn.c]: And this line moves too.

File: src/txn/txn.c [hunk 3]
```diff
@@ -1627,22 +1627,34 @@ __wt_verbose_dump_txn_one(WT_SESSION_IMPL *session, WT_TXN *txn)
 		break;
 	}
 	WT_RET(__wt_msg(session,
-	    "mod count: %u"
+	    "transaction id: %" PRIu64
+	    ", mod count: %u"
 	    ", snap min: %" PRIu64
 	    ", snap max: %" PRIu64
+	    ", no of named snapshots: %u"
```
Comment (sueloverso) [inline src/txn/txn.c]: This is not quite correct. I suggest just calling it `snapshot count`.

Comment (kommiharibabu) [reply src/txn/txn.c]: Corrected.

File: src/txn/txn.c (hunk 1 above)
Comment (kommiharibabu) [reply src/txn/txn.c]: Changed.

File: src/txn/txn.c (hunk 2 above)
Comment (kommiharibabu) [reply src/txn/txn.c]: changed.

**Pull request merged**